apps/server/lib/lexical/server/window.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Lexical.Server.Window do
```

Comment:
This being in the server app means we won't be able to use it in `remote_control`, which is where the vast majority of work happens. This might not be an issue, and I don't want to create a bunch of yak-shaving for this PR, but we might need to use a `RemoteControl.Dispatch` listener or handler to respond to messages coming from the `remote_control` project.

Reply:
Ah this is a really helpful comment. I was having trouble understanding what code should live in `remote_control` vs `server`. I don't see another instance of code in `server` subscribing to `RemoteControl.Dispatch` but I think that would be a valid thing to add in this PR if you wish? The ability to invoke logs and messages from remote_control could be needed eventually

Reply:
The dependencies in the sub apps go one way. Anyone can use common, proto and protocol are used by server, as is remote_control. `server` can use `remote_control`, but not vice-versa. 

There is a lot of code in `server` that subscribes to certain messages via dispatch, it's just done via `RemoteControl.Api.register_listener`. There are a bunch of things that do that at present. 

I don't think we have to drink the whole ocean with your first PR, we can add the ability of `RemoteControl` to send messages later.